3]. Competitive Programming/HackerRank/Problem Solving/Algorithms/Python/_01) A Very Big Sum.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Solution
![image](https://user-images.githubusercontent.com/60146338/122334420-50d25c80-cf57-11eb-8670-658272db7a0c.png)

Answer
![image](https://user-images.githubusercontent.com/60146338/122334450-6182d280-cf57-11eb-9a12-fbb9a7f55796.png)


Please write full code

Reply:
That's because you're supposed to just copy paste the function, not the entire code as the HackerRank template takes care of the input.